### ISSUE TITLE
GUI : Design : Server Configuration - HMC

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#1260]
## Stakeholders
**BMC SME**: @ratagupt 
**RAS SME**: Justin Thaler
**PHYP SME**: BJ Vette
**IBM i SME**: Mike Groeschel, Randy Tomfohrde
**Design Researcher**:
**UX Designer**:
**FED**:

## Overview
-  Show  HMC's Connected to BMC
- HMC  Configuration

## Use Case
TBD

## InVision Prototype
- [Iteration 0 - Low fidelity](https://ibm.invisionapp.com/share/Q6NZ13M3A5B#/319379893_01-Hmc-1)
- [Iteration 1 (Final)](https://ibm.invisionapp.com/share/Q6NZ13M3A5B#/319544180_01-Hmc-03)

## Design Issue (webui-vue)
TBD

## References/Resources
- [4.11.1 and 4.11.2 in eBMC Features worksheet](https://ibm.ent.box.com/file/455779315260)
- feature discovery folder: https://ibm.ent.box.com/folder/93346785327
- user research notes*: 
- user research synthesis: 

*This folder is restricted in accordance with GDPR guidelines.

**Comment by ryanarnell** [comment]
![02-hmc-1@2x](https://user-images.githubusercontent.com/4976631/73682814-c2717980-4686-11ea-83f6-1d3b0af452ea.png)


we tested our low fi prototype with users and this is what we have found-
Users would benefit from these fields

- HMC name
- Machine Type 
- Serial number 
- IP address 
- Status

Other Possible actions including being able to delete temporarily disconnected ones

**Comment by derick-montague** [comment]
Currently no backend work in progress. @ratagupt can you share the backend stories with us so we have a better idea of when work will begin?

**Comment by derick-montague** [comment]
to: @ratagupt 
cc: @gtmills @ryanarnell @mzipse 

1.  Is the work to show connected BMC's in progress? If so, what is the story so that we can link it to the Web UI?
1. Can you help us understand why the `machine type` and `serial number` can not be returned as two separate fields? It has been discussed that it will be returned from the API as 1 field formatted like `machine type*serial number`. What is the purpose for only returning one concatenated string?
1. Users are asking for the HMC names to be provided to the BMC. Is there a technical limitation that prevents us from providing this information to the user?

**Comment by derick-montague** [comment]
@ryanarnell is scheduling a meeting with @ratagupt  @gtmills and @derick-montague to discuss design needs and outstanding questions needed to unblock this story.

**Comment by derick-montague** [comment]
Follow up on Slack if needed: https://ibm-systems-power.slack.com/archives/C0Q6TQP5Z/p1583855789172000

**Comment by derick-montague** [comment]
Ratan is working with the community to get some items added. He is working on a unique identifier property that could be used for the HMC name. If that is not accepted by the community, we can look into getting the hostname from  this API as separate calls: 

http://man7.org/linux/man-pages/man3/getnameinfo.3.html

**Comment by ryanarnell** [comment]
![01-hmc-032@2x](https://user-images.githubusercontent.com/4976631/82055131-b03fa800-9685-11ea-8ce6-4eaae9019409.png)

According to the redfish changes we will only be able to show Machine type, serial number and IP address. The latest prototype reflects that.

**Comment by nicoleconser** [comment]
Guessing we need to revisit this to rename it to something more generic? @derick-montague @ParishrutB @yoshiemuranaka

**Comment by gtmills** [comment]
@nicoleconser Why not just show all the connections ?

**Comment by nicoleconser** [comment]
> @nicoleconser Why not just show all the connections ? Show all the sessions

Agreed - I assumed this page would show all client connections, not just the HMC. My question was more around - I don't think we can call this page "Hardware Management Consoles" since that's an IBM term (HMC).

**Comment by nicoleconser** [comment]
@derick-montague @yoshiemuranaka  Should we re-open this or create a new issue?

**Comment by derick-montague** [comment]
Let's open new and add this as a resource in the description.

**Comment by nicoleconser** [comment]
Created: https://github.com/ibm-openbmc/dev/issues/2483
